src/pilotdatasync-xp11.cpp

```diff
@@ -102,6 +108,64 @@ void dummy_key_handler(
     int losing_focus
 ) {}
 
+int runTCPThread(){
```

Comment:
The function has various integer returns, but do we have any particular meaning applied to each returned number?